graph/api/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't see a strong need for 3 (!) new modules here. Just put them into one? This doesn't cover that much ground.

Reply:
The idea with three modules is to allow one for Cypher querying, one for Graph Algorithms, and one parent to unite the both. Is there a large cost associated with additional modules?

Reply:
Just to add to his: We discussed the module architecture on the [SPIP design sketch document](https://docs.google.com/document/d/1Wxzghj0PvpOVu7XD1iA8uonRYhexwn18utdcTxtkxlI/edit#heading=h.om6xtdv3p3fn) and offline with @mengxr. We want to have the option for user applications to switch the engine implementation without changing the API. Furthermore, graph algorithms ported from GraphFrames/GraphX will live in an additional module that depends on `graph-api`.

Reply:
I don't feel strongly about it, but it seems like overkill for a small module that probably only has one implementation in the foreseeable future. I generally would have preferred (at least some of) this live outside Spark, but I agree to disagree